### PR TITLE
Adding simsopt package

### PR DESCRIPTION
**Pull request by mbkumar**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

simsopt package is hybrid python/C++ package.
@conda-forge-admin, please ping team

### CODE REVIEW

**Comment by landreman** [comment]
I confirm I am willing to be listed as a maintainer.

File: recipes/simsopt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hiddenSymmetries/simpsopt/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5201fcd8f53857791c2d8e18fe608845e4d18b6c25eac96625d47700a52df87f
+
+build:
+  number: 0
+  skip: "True  # [win, py<37]"
```
Comment (marcelotrevisani) [inline recipes/simsopt/meta.yaml:14]:
```suggestion
  skip: True  # [win or py<37]
```

Comment (mbkumar) [reply recipes/simsopt/meta.yaml]: Thank you! I was struggling with that for a while since the documentation was scarce on that point.

**Comment by mbkumar** [comment]
@marcelotrevisani, pinging for your review.

Thanks

**Comment by mbkumar** [comment]
@conda-forge-admin, please ping team

**Pull request closed without merging**